src/UnixTerminal.php

```diff
@@ -196,6 +196,22 @@ public function clearLine() : void
         $this->output->write(sprintf("\033[%dD\033[K", $this->getWidth()));
     }
 
+    /**
+     * Erase the entire current line
+     */
+    public function clearEntireLine() : void
```

Comment:
Would this just replace clear line, not sure what the difference is between the two 

Maybe the current clear line only does from cursor to the right the same amount of cols as the width :thinking: